chain/chain/src/metrics.rs

```diff
@@ -18,6 +18,13 @@ pub static BLOCK_PROCESSED_SUCCESSFULLY_TOTAL: Lazy<IntCounter> = Lazy::new(|| {
 pub static BLOCK_PROCESSING_TIME: Lazy<Histogram> = Lazy::new(|| {
     try_create_histogram("near_block_processing_time", "Time taken to process blocks").unwrap()
 });
+pub static BLOCK_PROCESSING_SUCCESSFULLY_TIME: Lazy<Histogram> = Lazy::new(|| {
+    try_create_histogram(
+        "near_block_successful_processing_time",
```

Comment:
Var name `BLOCK_PROCESSING_SUCCESSFULLY_TIME` should probably be consistent with counter name `near_block_successful_processing_time`